css-responsive/css-responsive_playground/css/styles.css

```diff
@@ -71,26 +71,58 @@ h1 {
 
   width: 100px;
   height: 100px;
+
+  @media (min-width: 600px) {
+    width: 200px;
+}
+
+@media (min-width: 1200px) {
+  width: 400px;
+}
 }
 
 .box--three {
   background-color: lightpink;
 
   width: 50px;
   height: 250px;
+
+  @media (orientation: landscape) {
+    width: 250px;
+    height: 50px;
+  }
 }
 
 .box--four {
   background-color: lightgray;
 
   width: 100px;
   height: 100px;
+
+  @media (prefers-color-scheme: dark) {
+    background-color: aqua;
+  }
 }
 
 .box--five {
   width: 100%;
+
+  
+
 }
 
 .box--five img {
   width: 100%;
+
+  <picture>
```

Comment:
You dont need this HTML code in your CSS

Reply:
Ah, yes, I overlooked that snippet! I intented to delete all that html in the css. Done now! Thank you!